### PR TITLE
Varun

### PR DESCRIPTION
**Pull request by Techvarun1991**
I have done the Issues related API integration please check this out !!

### CODE REVIEW

File: src/Components/AllIssues.js [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+  var payload = {project_id: ProjectID};
+  console.log(payload);
+  useEffect(()=>{
+    AuthenticationService.allIssues(payload).then((response)=>{
+      console.log(response.data);
+    }).catch((error)=>{
+      console.log(error);
+    })
+  },[])
+
```
Comment (anupkore) [inline src/Components/AllIssues.js:35]: You can remove this commented code if not required in future

File: src/Components/IssueCard.js
Comment (anupkore) [inline src/Components/IssueCard.js:1]: Remove commented code

File: src/Components/SideBar.js [hunk 2]
```diff
@@ -44,6 +40,17 @@ export default function SideBar(props) {
     }
   };
```
Comment (anupkore) [inline src/Components/SideBar.js:42]: This UseEffect is not in use

File: src/Components/UpdateProjectForm.js [hunk 3]
```diff
@@ -56,30 +59,58 @@ export default function UpdateProjectForm(props) {
     setDescription(event.target.value);
   }
 
+  // function handleUpdateProject(event) {
```
Comment (anupkore) [inline src/Components/UpdateProjectForm.js:62]: This function is also not in use

**Review by anupkore — CHANGES_REQUESTED** [review]

File: src/Components/UpdateProjectForm.js (hunk 3 above)
Comment (Techvarun1991) [reply src/Components/UpdateProjectForm.js:62]: ok i will remove it

File: src/Components/AllIssues.js (hunk 1 above)
Comment (Techvarun1991) [reply src/Components/AllIssues.js]: I have remove the unwanted Part

File: src/Components/IssueCard.js
Comment (Techvarun1991) [reply src/Components/IssueCard.js:1]: Removed

**Review by anupkore — APPROVED** [review]

**Pull request merged**